.changes/next-release/bugfix-04f91235-8b99-45c3-a4bd-096c530739af.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Security Scan: Improved method of applying security fixes"
```

Comment:
nit: Is the effect for the user going to be the same, if yes, then do we need the changelog?

Reply:
The effect is the same, the difference is that it is less likely to fail with this change. Do you suggest a different changelog message?